llvm/lib/Target/AVR/MCTargetDesc/AVRAsmBackend.cpp

```diff
@@ -32,7 +32,7 @@ namespace adjust {
 using namespace llvm;
 
 static void unsigned_width(unsigned Width, uint64_t Value,
-                           std::string Description, const MCFixup &Fixup,
+                           const std::string &Description, const MCFixup &Fixup,
```

Comment:
This should be using StringRef, and the code below should be using Twine instead of building the string itself